lib/workers/miq_worker_types.rb

```diff
@@ -47,7 +47,6 @@
   "ManageIQ::Providers::Openstack::InfraManager::RefreshWorker"                 => %i(manageiq_default),
   "ManageIQ::Providers::Openstack::NetworkManager::EventCatcher"                => %i(manageiq_default),
   "ManageIQ::Providers::Openstack::NetworkManager::MetricsCollectorWorker"      => %i(manageiq_default),
-  "ManageIQ::Providers::Openstack::NetworkManager::RefreshWorker"               => %i(manageiq_default),
```

Comment:
we should probably do the same for "ManageIQ::Providers::StorageManager::CinderManager::RefreshWorker" ? Or do we support that as a separate manager now?

Reply:
hm also for ManageIQ::Providers::StorageManager::SwiftManager::RefreshWorker, since all of those are returned as ```.child_managers```, right?

Reply:
They probably should be run in the same worker since I think the only use case we support right now is as part of an Openstack provider afaik, but I know that Cinder and Swift have been kept separate for some reason.

Reply:
right, so right now, in the OpenStack PR, you will be doing all the work of ```.child_managers``` in that 1 worker. So the other workers need to be removed.

We will need to revisit that, once we have separate Swift (SwiftStack) and Cinder (there was something like CinderStack in progress, like year ago?) managers

Reply:
Cool.

Reply:
Just to confirm, it isn't possible to run a Swift or Cinder StorageManager with _any_ other provider?

Reply:
As far as I know there's no way to add Swift or Cinder as stand alone providers, and I'm not aware of any other provider other than the Openstack Cloud Provider that they're used as part of in ManageIQ. @Ladas might want to confirm that though.

Reply:
The plan was to run them separately at some point. But looking at:
https://github.com/Ladas/manageiq/blob/742b0d68bf0d0a251fe226f81fde719dfac1086a/app/models/manageiq/providers/storage_manager/cinder_manager.rb#L31
and
https://github.com/Ladas/manageiq/blob/742b0d68bf0d0a251fe226f81fde719dfac1086a/app/models/manageiq/providers/storage_manager/swift_manager.rb#L27

It seems like the parent manager is still a requirement

Reply:
If we're going to tie these to Openstack I wonder if we should just move them to the manageiq-providers-openstack repo?  @Fryguy ?

Reply:
@agrare I am not sure why those are not under OpenStack tbh.

Reply:
Okay if we end up supporting these as standalone we can change the [`combined_managers`](https://github.com/ManageIQ/manageiq-providers-openstack/pull/154/files#diff-aede0abf33416e3c7f148d3fba1e39ceR17) to exclude storage managers